src/components/services/ActionQueueService.ts

```diff
@@ -112,14 +112,10 @@ export const ActionQueueService = () => {
         // Set program state to executing
         await updateProgramState(state, true)
 
-        if (program.mockMode) {
-          const { nextState } = await mockActionProgram(executionContext, program, state)
-          // Update program state
-          await updateProgramState(nextState, false)
-          return
-        }
+        // Use mock execution function if program is marked as mockMode
```

Comment:
General comment for this commit: Let's create a task for reimplementing the mock mode to restrict it to the appropriate areas in the runtime

Reply:
I agree